core/primitives/src/runtime/config.rs

```diff
@@ -42,13 +52,45 @@ lazy_static::lazy_static! {
     static ref LOWER_STORAGE_COST_CONFIG: Mutex<Option<Arc<RuntimeConfig>>> = Mutex::new(None);
 }
 
+#[cfg(feature = "protocol_feature_add_account_versions")]
+lazy_static::lazy_static! {
+    static ref STORAGE_USAGE_DELTA_FROM_FILE: HashMap<AccountId, u64> = RuntimeConfig::read_storage_usage_delta();
+    static ref EMPTY_STORAGE_USAGE_DELTA: HashMap<AccountId, u64> = HashMap::new();
+    // static ref STORAGE_USAGE_DELTA_CSV: str = include_str!("../../res/storage_usage_delta.csv");
+}
+
+#[cfg(feature = "protocol_feature_add_account_versions")]
+lazy_static_include::lazy_static_include_str! {
+    STORAGE_USAGE_DELTA_CSV => "res/storage_usage_delta.csv"
+}
```

Comment:
This looks like a hotfix for mainnet. Is it? Is it so messed up that there is no other way but to have a handcrafted list of accounts?

Reply:
It is not handcrafted, but generating it on the fly is not viable. I'll add description